ldes-server-infra-mongo/mongo-repository/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/infra/mongo/member/MemberMongoRepository.java

```diff
@@ -30,14 +27,7 @@ public MemberMongoRepository(final LdesMemberEntityRepository repository, MongoT
 
 	@Override
 	public Member saveLdesMember(Member member) {
-		Query query = new Query();
-		query.addCriteria(Criteria.where("_id").is(member.getLdesMemberId()));
-		Update update = new Update();
-		StringWriter outputStream = new StringWriter();
-		RDFDataMgr.write(outputStream, member.getModel(), Lang.NQUADS);
-		String ldesMemberString = outputStream.toString();
-		update.set("model", ldesMemberString);
-		mongoTemplate.upsert(query, update, LdesMemberEntity.class);
+		repository.save(LdesMemberEntity.fromLdesMember(member));
```

Comment:
This is fine for me, but I thought we used the template for performance reasons?

Reply:
That's correct. Template was used so that the attributes of LdesMemberEntity could be updated separately (this enabled fragmentation to already start even if the member was not saved). The reason why I removed that here, is because we're going to rethink the whole ingest-fragmentation anyway and it feels icky to use mongoTemplate to update almost all attributes of a member at the same time (model, versionOf, timestamp, id).